src/wmtk/utils/getRSS.cpp

```diff
@@ -31,13 +33,15 @@
 #error "Cannot define getPeakRSS( ) or getCurrentRSS( ) for an unknown OS."
 #endif
 
+#include "getRSS.h"
+
 
 /**
  * Returns the peak (maximum so far) resident set size (physical
  * memory use) measured in bytes, or zero if the value cannot be
  * determined on this OS.
  */
-size_t getPeakRSS()
+size_t wmtk::getPeakRSS()
```

Comment:
one sorta ugly but sorta nice thing is that in cpp you can just say `using namespace wmtk` and it'll auto-figure out that you're defining the declaration `wmtk::getPeakRSS`

Reply:
For this I prefer to wrap everything in a `namespace wmtk { .... }` surrounding the entire .cpp ....

Reply:
I used the `namespace wmtk {...}` version. Just writing `using namespace wmtk` seems like it does not work for MSVC.